### PR TITLE
77349- Separate new/continue

### PR DESCRIPTION
**Pull request by will-hope98**

### CODE REVIEW

File: src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs [hunk 1]
```diff
@@ -260,25 +265,33 @@ public async Task HandleAsync_GivenOrganisationHasNoDirectRegistrant_Organisatio
...
-            A.CallTo(() => directRegistrant.Id).Returns(Guid.NewGuid());
+            var directRegistrantId = Guid.NewGuid();
+
+            A.CallTo(() => directRegistrant.Id).Returns(directRegistrantId);
             A.CallTo(() => directRegistrant.OrganisationId).Returns(organisation.Id);
 
-            var directRegistrants = new List<DirectRegistrant> { directRegistrant };
+            A.CallTo(() => directRegistrant.DirectProducerSubmissions).Returns(new List<DirectProducerSubmission>
+            {
+                new DirectProducerSubmission { ComplianceYear = DateTime.UtcNow.Year }
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs:281]: @will-hope98  We might get issues with the test running in new years. Think we need to setup the systemDataDataAccess to return a known date with compliance year  and use that as the compliance year.

File: src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs [hunk 2]
```diff
@@ -260,25 +265,33 @@ public async Task HandleAsync_GivenOrganisationHasNoDirectRegistrant_Organisatio
...
 
             var message = new GetOrganisationInfo(organisationId);
 
+            // Act
             var result = await handler.HandleAsync(message);
 
+            // Assert
             result.HasDirectRegistrant.Should().BeTrue();
-            result.DirectRegistrantId.Should().Be(directRegistrant.Id);
+            result.DirectRegistrants.Should().ContainSingle(dr => dr.DirectRegistrantId == directRegistrantId);
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs:294]: @will-hope98  need an assert to check the submission started

File: src/EA.Weee.Web/Areas/Scheme/Controllers/HomeController.cs [hunk 3]
```diff
@@ -138,7 +142,19 @@ internal async Task<List<string>> GetActivities(Guid pcsId, OrganisationData org
                 if (organisationDetails.HasDirectRegistrant)
                 {
                     activities.Add(ProducerSubmissionConstant.HistoricProducerRegistrationSubmission);
-                    activities.Add(ProducerSubmissionConstant.NewContinueProducerRegistrationSubmission);
+
+                    foreach (var registrant in organisationDetails.DirectRegistrants)
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Controllers/HomeController.cs:146]: @will-hope98  think for the time being we just  get the first item here, which will get updated in another story. As it is we have potential to add duplicate options.

Comment (will-hope98) [reply src/EA.Weee.Web/Areas/Scheme/Controllers/HomeController.cs]: Done

File: src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs (hunk 2 above)
Comment (will-hope98) [reply src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs:298]: Done

File: src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs (hunk 1 above)
Comment (will-hope98) [reply src/EA.Weee.RequestHandlers.Tests.Unit/Organisations/OrganisationByIdHandlerTests.cs]: Done

**Pull request merged**